index.html

```diff
@@ -130,26 +145,31 @@ <h6 class="client">Canopy</h6>
                     <li class="list-item"><button type="button" class= "btn btn-tags"> javaScript</button></li>
                 </ul>
                 <div class="action">
-                    <button  type="button" disabled class="btn normal-b"> See project</button>
+                    <button  type="button" class="btn normal-b"> See project</button>
                 </div>
             </div>
         </div>
         <div class="about-myself">
-            <h2 class= "title-about-myself">About myself</h2>
-            <p class="intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-            <h6 class="connect">let's connect</h6>
-            <ul class="networks-about">
-                <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
-                <li ><a href=""><img src="img/Linkedin icon.png" alt=""></a></li>
-                <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
-                <li><a href=""><img src="img/Twitter.png" alt="20px"></a></li>
-                <li><a href=""><img src="img/M.png" alt="20px"></a></li>
-            </ul>
-            <div class="about-action">
-                <button type="button" disabled class="btn normal-b">Get my resume</button>
-            </div>
+            <div class="left-block-about">
+                <h2 class= "title-about-myself">About myself</h2>
+                <p class="intro-about intro">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+                <h6 class="connect-about connect">let's connect</h6>
+                <ul class="networks-about">
+                    <li><a href=""><img src="img/GitHub.png" alt="20px"></a></li>
+                    <li ><a href=""><img src="img/Linkedin icon.png" alt=""></a></li>
+                    <li><a href=""><img src="img/Peace.png" alt="20px"></a></li>
+                    <li><a href=""><img src="img/Twitter.png" alt="20px"></a></li>
+                    <li><a href=""><img src="img/M.png" alt="20px"></a></li>
+                </ul>
+                <div class="about-action">
+                    <button type="button" class="btn normal-b">Get my resume</button>
+                </div>
+            </div>    
             <div class="languages">
-                <h3 class="languages-title">Languages</h3>
+                <div class="languages-cont">
+                    <h3 class="languages-title">Languages</h3>
+                    <i class="fa-solid fa-chevron-down"></i>
```

Comment:
A chevron is supposed to appear here (just on the opposite side of `Languages`) if I'm not mistaken 🤔 but it's not appearing in the mobile version. Take a look below 👇🏽 

![Screenshot 2023-02-24 at 08 35 01](https://user-images.githubusercontent.com/45739963/221191978-33d36f2f-fea7-42fe-9dc6-1ab9776b9c3b.png)

Reply:
@didierganthier hi! thank you for your review! 
I don't know what it's happenning because when i see the project on the live server everything its right. do you have any suggestion on what it could be?

Reply:
Hi @federicaulzurrun, I thought that and I tried calling you on Zoom to be sure of that. Not sure what is causing the issue but I'm available in a bit to get on a call for 30 mins if you wish. Another suggestion, if it's working perfectly on your side, record a demo video and add it to the description. Anyway, feel free to message me on Slack or tag me here if you have any questions. 😉